Source/ee/IPU.cpp

```diff
@@ -734,7 +734,9 @@ void CIPU::COUTFIFO::RequestGrow(unsigned int size)
 	while(m_alloc <= (size + m_size))
 	{
 		m_alloc += GROWSIZE;
-		m_buffer = reinterpret_cast<uint8*>(realloc(m_buffer, m_alloc));
```

Comment:
Indeed, the code doesn't handle the case where `realloc` fails.

But your proposed change seems dangerous. If `realloc` fails, `m_buffer` is not updated while `m_alloc` is. Thus, the rest of the code will assume `m_buffer` points to a memory area of `m_alloc` size when it's not, causing potential memory overruns.

Unless there's a specific reason to address the failing `realloc` case, I wouldn't touch this code. If the allocation fails, there's not much we can do anyways (besides crashing).